pypesto/objective/petab_import.py

```diff
@@ -388,16 +287,17 @@ def create_objective(self,
         return obj
 
     def create_problem(self, objective):
-        problem = Problem(objective=objective,
-                          lb=self.petab_problem.lb,
-                          ub=self.petab_problem.ub,
-                          x_fixed_indices=self.petab_problem.x_fixed_indices,
-                          x_fixed_vals=self.petab_problem.x_fixed_vals,
-                          x_names=self.petab_problem.x_ids)
+        problem = Problem(
+            objective=objective,
+            lb=self.petab_problem.lb_scaled,
+            ub=self.petab_problem.ub_scaled,
+            x_fixed_indices=self.petab_problem.x_fixed_indices,
+            x_fixed_vals=self.petab_problem.x_nominal_fixed_scaled,
```

Comment:
```suggestion
            x_fixed_vals=self.petab_problem.x_fixed_values,
```

Reply:
We require the values to be scaled, so if I am not missing something, we need ...scaled... here, right?

Reply:
`petab_problem.x_nominal_fixed_scaled` doesn't exist anymore

Reply:
oh it does on develop :( my bad ...